.github/workflows/cd.yml

```diff
@@ -0,0 +1,56 @@
+name: Continuous Deployment
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  docker_push:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: tactilenews/hundred-eyes:latest
+      - name: Image digest
+        run: echo ${{ steps.docker_build.outputs.digest }}
+
+  deploy:
+    needs: [docker_push]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install Ansible galaxies
+        run: |
+          ansible-galaxy collection install community.general
+          ansible-galaxy collection install devsec.hardening
+          ansible-galaxy install geerlingguy.docker
+          ansible-galaxy install geerlingguy.pip
+          ansible-galaxy install gantsign.inotify
+
+      - name: Run playbook
+        uses: dawidd6/action-ansible-playbook@v2
```

Comment:
Is it really necessary that we use a third-party action to run the playbook? Couldn’t we achieve the same using plain `run` steps? If you really want to use that action, could you pin it to a specific commit hash?